micro-rdk/src/common/grpc.rs

```diff
@@ -1357,7 +1356,7 @@ impl<'a> GrpcServerInner<'a> {
             .get_camera_by_name(req.name)
             .ok_or(GrpcError::RpcUnavailable)?;
 
-        let mut buffer = RefCell::borrow_mut(&self.buffer).split_off(0);
+        let mut buffer = BytesMut::with_capacity(CAMERA_BUFFER_SIZE);
```

Comment:
shouldn't this use the new allocator?

Reply:
I didn't bother because it would simply change immediately in the next PR